### ISSUE TITLE
in_tail: Choose multiple patterns for Path

### ISSUE THREAD

**Issue by alitoufighi** [fluent/fluent-bit#1508]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
In documentation, it is said that we can define the `Path` using `common wildcards`. But it is not clear what wildcards are considered `common`. We are limited to only one pattern, but in `Exclude_Path` section, multiple patterns are supported. It would be nice if we can choose multiple values (comma separated) for `Path` to select logs from.

Consider I want to collect all logs within `foo` and `bar` namespace. I prefer to have option to choose them like this:
```
[INPUT]
        Name              tail
        Tag               kube.*
        Path              /var/log/containers/*_foo_*.log, /var/log/containers/*_bar_*.log, 
```

Or supporting more wildcards to make people able to use this:
```
[INPUT]
        Name              tail
        Tag               kube.*
        Path              /var/log/containers/*_{foo,bar}_*.log 
```

**Additional context**
I want to ship logs of my pods in two namespaces of my Kubernetes cluster with Fluent Bit. But there is no documented way to direct the `path` value of `tail` input to get only logs of these namespaces. The only workaround I came with was using this wildcard `*_[fb]*_*` to select `foo` and `bar` namespaces; which is not a good selector.

**Comment by donbowman** [comment]
you can either:
- create 2 inputs, one for each namespace, with appropriate wildcard
- pick up all logs, and filter later.

both will work.

**Comment by rmacian** [comment]
Pure Fluentd works as described by the op, and I was expecting the same:

https://docs.fluentd.org/input/tail#path

but in fluent-bit is seems is not recognized:

```
[2019/10/31 13:19:53] [debug] [in_tail] scanning path /var/log/containers/*namespace1*.log,/var/log/containers/*_namespace2_*.log
[2019/10/31 13:19:53] [debug] [in_tail] Cannot read info from: /var/log/containers/*_namespace1_*.log,/var/log/containers/*_namespace2_*.log

```

**Comment by edsiper** [comment]
labeled as an enhancement request, path support only one entry, agreed, it needs to be improved.

**Comment by eldada** [comment]
For the sake of simplicity, I think a comma delimited list is a great idea. Also aligns with the `Exclude_Path` key...

**Comment by jcuna** [comment]
To add to this, the documentation says `The tail input plugin allows to monitor one or several text files. It has a similar behavior like tail -f shell command.` So I automatically thought about specifying multiple paths. Please add this feature!

**Comment by edsiper** [comment]
FYI: I am working on this, I have a functional code:

```
$ bin/fluent-bit -i tail -p path_key=file -p "path=a.log, b.log, [12].log, *.log" -o stdout -f 1
Fluent Bit v1.6.0
* Copyright (C) 2019-2020 The Fluent Bit Authors
* Copyright (C) 2015-2018 Treasure Data
* Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
* https://fluentbit.io

[2020/07/15 16:36:19] [ info] [engine] started (pid=346989)
[2020/07/15 16:36:19] [ info] [storage] version=1.0.4, initializing...
[2020/07/15 16:36:19] [ info] [storage] in-memory
[2020/07/15 16:36:19] [ info] [storage] normal synchronization mode, checksum disabled, max_chunks_up=128
[2020/07/15 16:36:19] [ info] [sp] stream processor started
[2020/07/15 16:36:19] [ info] inotify_fs_add(): inode=55716663 watch_fd=1 name=a.log
[2020/07/15 16:36:19] [ info] inotify_fs_add(): inode=55716671 watch_fd=2 name=b.log
[2020/07/15 16:36:19] [ info] inotify_fs_add(): inode=55716673 watch_fd=3 name=1.log
[2020/07/15 16:36:19] [ info] inotify_fs_add(): inode=55716674 watch_fd=4 name=2.log
[0] tail.0: [1594852579.041723729, {"file"=>"a.log", "log"=>"a"}]
[1] tail.0: [1594852579.041747351, {"file"=>"b.log", "log"=>"b"}]
[2] tail.0: [1594852579.041749952, {"file"=>"1.log", "log"=>"1"}]
[3] tail.0: [1594852579.041751872, {"file"=>"2.log", "log"=>"2"}]

```

**Comment by raine** [comment]
Fixed in v1.5.1.

https://fluentbit.io/announcements/v1.5.1/

**Comment by alitoufighi** [comment]
That's great!
I think we can happily close this issue now.
